cypress/integration/E2E.spec.ts

```diff
@@ -1,4 +1,5 @@
 /// <reference types="cypress" />
+import { HAITTA_INDEX_TYPE, HaittaIndexData } from '../../src/domain/common/haittaIndexes/types';
```

Comment:
Should we get rid off these Cypress integration tests? They are not used atm, right? And the plan was to use Playwright in E2E testing (when we have time or someone else comes to do it for us)? Are there any benefits of keeping these files in the repo?

Reply:
Yes, good idea. I was thinking of doing it in its own PR.

Reply:
Yes, that is a good plan.